src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -243,3 +249,59 @@ def test_wait_for_event_with_timeout():
         with session.initiate():
             session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE, 0.5)
 
+
+def test_commit(single_channel_session):
+    non_default_current_limit = 0.00021
+    single_channel_session.current_limit = non_default_current_limit
+    single_channel_session.commit()
+    assert single_channel_session.current_limit == non_default_current_limit
+
+
+# TODO (bhaswath): Enable test after Pull request #467 is merged, which will enable the export signal enum
+'''
+def test_export_signal(single_channel_session):
+    expected_trigger_terminal = "//Engine0/MeasureTrigger"
+    single_channel_session.export_signal(nidcpower.Event.SOURCE_COMPLETE, expected_trigger_terminal)
+    assert expected_trigger_terminal == single_channel_session.source_complete_event_output_terminal
+'''
+
+
+def test_configure_digital_edge_measure_trigger_error(single_channel_session):
+    try:
+        single_channel_session.configure_digital_edge_measure_trigger("invalid string")
+        with single_channel_session.initiate():
+            single_channel_session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE, 0.5)
+        assert False
+    except nidcpower.Error as e:
+        assert e.code == -1074097882  # Error : Invalid property value
+        assert e.description.find('Requested value is not a supported value for this property.') != -1
+
+
+def test_configure_digital_edge_pulse_trigger_error(single_channel_session):
+    try:
+        single_channel_session.configure_digital_edge_pulse_trigger("invalid string")
+        assert False
+    except nidcpower.Error as e:
+        assert e.code == -1074135023  # Error : Unsupported function
+        assert e.description.find('Function or method not supported.') != -1
+
+
+def test_configure_digital_edge_sequence_advance_trigger(single_channel_session):
+    expected_trigger_terminal = "//PXI_Trig0"
```

Comment:
Why the double slash?
Is this truly what the driver is returning? It appears to be a bug!